src/js/components/triviaQuestions.js

```diff
@@ -0,0 +1,17 @@
+import {createQuestion} from '../view/createQuestion';
+import {Pregunta} from '../models/Pregunta';
+
+export const triviaContainer = document.querySelector('.pokemonTrivia');
+triviaContainer.addEventListener('click',(event)=>{
+
+    let questionTitle = document.querySelector('.pregunta1__title')
```

Comment:
try to not hardcode string they can change and they are easy to lose
try 
```
 questions: {
   one: 'Muy bien respuesta correcta!, fue muy facil sigamos con la siguiente",'Cual de est',
    two: '',
  ....
}
```
they will not change also this is a strategy used when you will do multilanguage if later one we want texts in spanish and english they will be stored in a same place

Reply:
done